cli/src/main/resources/help/ls.adoc

```diff
@@ -13,6 +13,24 @@ SYNOPSIS
 *ls* ['PATH']
 
 
+OPTIONS
+-------
+
+*-f, --format*='[NAMES|VALUES|FULL]'::
+This option only applies when listing caches.
+* `NAMES`: only show the keys
+* `VALUES`: show the keys and values
+* `FULL`: show keys, values and metadata
+
+*-l*::
+This option only applies when listing caches.
+Shortcut for `-f FULL`.
```

Comment:
not sure how useful this is, because it looks like the shorter version of `--limit`.

Reply:
I know, but I wanted to make it more similar to the Unix `ls -l`. I could rename limit to `-m/--max-items`

Reply:
I've gone for -m